data/campaigns/Eastern_Invasion/scenarios/03_An_Unexpected_Appearance.cfg

```diff
@@ -1,167 +1,173 @@
 #textdomain wesnoth-ei
 
+#
+# 2 options - west or east
+#
+# west is easier: flat terrain, and you can win before Mal-Kallat's forces catch up to you
+#    but leads to the low-reward Elven Interlude
+# east is harder: fighting bats/ghosts in difficult terrain, and Mal-Skraat's forces will catch up to you
+#    but leads to the high-reward Bitter Swamp and Mal-Ravanal's Capital
+#
+
+#define TURNS
```

Comment:
It's also used in a `name=side 1 turn {TURNS} end` event, so I used a macro so that it would be impossible to change one without the other.  Could be done in different ways as well.

Not good that it overwrites the built-in `TURNS` macro; I hadn't realized that.

Reply:
You're also not undeffing it at the bottom of the file.

Is there a reason why `name=time over` does not work for your desired use-case, though?